packages/buefy-next/src/components/rate/Rate.vue

```diff
@@ -89,14 +96,15 @@ export default {
         showMe() {
             let result = ''
             if (this.showScore) {
-                result = this.disabled ? this.modelValue : this.newValue
-                if (result === 0) {
+                result = this.disabled ? this.modelValue.toString() : this.newValue.toString()
+                if (Number(result) === 0) {
                     result = ''
                 } else {
-                    result = new Intl.NumberFormat(this.locale).format(this.modelValue)
+                    result = new Intl.NumberFormat(this!.locale)
+                        .format(this.modelValue).toString()
```

Comment:
`toString` should not be necessary here.

Reply:
The reason in my mind why I have isString there is because I thought the result variable was of type string?

Reply:
> The reason in my mind why I have isString there is because I thought the result variable was of type string?

Yes, and the `format` method of `Intl.NumberFormat` returns a string.

Reply:
Got it!